ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/home/view/HomeView.kt

```diff
@@ -78,7 +86,31 @@ fun Home(
 
 @Composable
 private fun HomeTopAppBar() {
-    SmallTopAppBar(title = { Text(text = stringResource(id = R.string.app_name)) })
+    SmallTopAppBar(
+        title = { Text(text = stringResource(id = R.string.app_name)) },
+        actions = {
+            DebugMenu()
```

Comment:
I should hide this when using Firebase Test Lab (for robo tests) and also for our screenshot tests.